frontend/src/components/gamefield/Leaderboard.js

```diff
@@ -0,0 +1,37 @@
+import "./Leaderboard.css";
+
+const Leaderboard = (props) => {
+
+  const { leaderboardArray, localPlayer } = props;
+  if (leaderboardArray.length === 0) {
+    return null;
+  }
+
+  leaderboardArray.sort((a, b) => b[1] - a[1]);
```

Comment:
Nem illik módosítani a props értékeit, mert ha valaki meghívja a Leaderboard függvényt (pl. rendereli), akkor nem számít arra, hogy ez majd sorbarendezi azt.
csinálj egy másolatot és dolgozz azzal:
https://www.geeksforgeeks.org/how-to-copy-array-by-value-in-javascript/